DuckDuckGo/AppConfigurationFetch.swift

```diff
@@ -108,6 +169,8 @@ class AppConfigurationFetch {
                 store.foregroundNoDataCount += 1
             }
         }
+
+        store.lastConfigurationRefreshDate = Date()
```

Comment:
I'm currently looking into how this function behaves if you have a background task which tries to run after the device has been rebooted and hasn't yet been unlocked, as it modifies `UserDefaults`.